apps/accounts/forms.py

```diff
@@ -93,9 +95,24 @@ def save(self, commit=True):
         user.set_password(password)
         if commit:
             user.save()
+            self.grant_organisation(user)
             self.save_m2m()
         return user
 
+    def grant_organisation(self, user):
+
+        email_domain = user.get_email_domain()
+
+        if Organisation.objects.filter(email__icontains=email_domain).exists():
+
+            organisation = Organisation.objects.get(email__icontains=email_domain)
+            if organisation in self.cleaned_data['chosen_organisations']:
+                org_email_domain = organisation.get_email_domain()
+                if org_email_domain == email_domain:
+                    user.is_active = True
```

Comment:
Slightly unsure on this line.

Unless we send out an email - we can't guarantee this?

Reply:
We can't Alex! :)) Let's probably remove `is_active=Tru` and let's wait what client says?

Reply:
👍